spec/policy_loader_spec.lua

```diff
@@ -0,0 +1,24 @@
+local _M = require 'apicast.policy_loader'
+
+describe('APIcast Policy Loader', function()
+
+  describe('.call', function()
```

Comment:
I think we need tests that show that the code works correctly when it loads two different versions of the same policy. I guess that testing this it's a bit difficult, because we'll need, at least, to create some test policies in the `fixtures` folder.

Maybe worth taking a look whether injecting `apicast_dir` would make that test easier or clearer.